packages/headless/src/controllers/controller/error-payload.ts

```diff
@@ -1,16 +1,3 @@
-export interface ErrorPayload {
-  /**
-   * The HTTP status code.
-   */
-  statusCode: number;
+import {SearchAPIErrorWithStatusCode} from '../../api/search/search-api-error-response';
 
-  /**
-   * The error message.
-   */
-  message: string;
-
-  /**
-   * The error type.
-   */
-  type: string;
-}
+export type ErrorPayload = SearchAPIErrorWithStatusCode;
```

Comment:
By making ErrorPayload a type-alias, the properties can no longer be properly extracted.